
Log Entry : 
10/18/2017 14:56:48
Error in: . Error Message:Microsoft.Reporting.WebForms.ReportServerException: No se encuentra el elemento '/Pronto informes/Williams - Controles Diarios'. (rsItemNotFound)
   en Microsoft.Reporting.WebForms.ServerReportSoapProxy.OnSoapException(SoapException e)
   en Microsoft.Reporting.WebForms.Internal.Soap.ReportingServices2005.Execution.RSExecutionConnection.ProxyMethodInvocation.Execute[TReturn](RSExecutionConnection connection, ProxyMethod`1 initialMethod, ProxyMethod`1 retryMethod)
   en Microsoft.Reporting.WebForms.Internal.Soap.ReportingServices2005.Execution.RSExecutionConnection.LoadReport(String Report, String HistoryID)
   en Microsoft.Reporting.WebForms.ServerReport.EnsureExecutionSession()
   en Microsoft.Reporting.WebForms.ServerReport.SetParameters(IEnumerable`1 parameters)
   en CartaDePorteManager.RebindReportViewer_ServidorExcel(ReportViewer& oReportViewer, String rdlFile, IEnumerable`1 parametros, String& ArchivoExcelDestino, Boolean bDescargaHtml) en C:\bdl\pronto\BussinessLogic\ManagerDebug\CartaDePorteManager.vb:línea 5193
__________________________

Log Entry : 
10/18/2017 17:32:19
Error in: . Error Message:Por generar el archivo C:\Users\Administrador\Downloads\Informe18oct.2017_173201.xls
__________________________

Log Entry : 
10/18/2017 17:32:19
Error in: . Error Message:System.IO.DirectoryNotFoundException: No se puede encontrar una parte de la ruta de acceso 'C:\Users\Administrador\Downloads\Informe18oct.2017_173201.xls'.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   en System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   en System.IO.FileStream..ctor(String path, FileMode mode)
   en CartaDePorteManager.RebindReportViewer_ServidorExcel(ReportViewer& oReportViewer, String rdlFile, String strSQL, String SC, Boolean bDescargaExcel, String& ArchivoExcelDestino, String titulo, Boolean bDescargaHtml) en C:\bdl\pronto\BussinessLogic\ManagerDebug\CartaDePorteManager.vb:línea 5589
__________________________

Log Entry : 
10/18/2017 17:41:52
Error in: . Error Message:Por generar el archivo C:\Users\Administrador\Downloads\Informe18oct.2017_174121.xls
__________________________

Log Entry : 
10/18/2017 17:42:48
Error in: . Error Message:Por generar el archivo C:\Users\Administrador\Downloads\Informe18oct2017_174121.xls
__________________________

Log Entry : 
10/18/2017 17:44:07
Error in: . Error Message:Por generar el archivo C:\Users\Mariano\Downloads\Informe18oct.2017_174345.xls
__________________________

Log Entry : 
10/18/2017 17:44:12
Error in: . Error Message:Archivo generado C:\Users\Mariano\Downloads\Informe18oct.2017_174345.xls
__________________________

Log Entry : 
10/18/2017 19:15:57
Error in: . Error Message:Por generar el archivo C:\Users\Mariano\Downloads\Informe18oct.2017_191529.xls
__________________________

Log Entry : 
10/18/2017 19:15:57
Error in: . Error Message:Archivo generado C:\Users\Mariano\Downloads\Informe18oct.2017_191529.xls
__________________________

Log Entry : 
10/18/2017 19:39:42
Error in: . Error Message:Por generar el archivo C:\Users\Mariano\Downloads\Informe18oct.2017_193930.xls
__________________________

Log Entry : 
10/18/2017 19:39:42
Error in: . Error Message:Archivo generado C:\Users\Mariano\Downloads\Informe18oct.2017_193930.xls
__________________________

Log Entry : 
10/18/2017 19:48:06
Error in: . Error Message:Por generar el archivo C:\Users\Mariano\Downloads\Informe18oct.2017_194041.xls
__________________________

Log Entry : 
10/18/2017 19:48:06
Error in: . Error Message:Archivo generado C:\Users\Mariano\Downloads\Informe18oct.2017_194041.xls
__________________________

Log Entry : 
10/18/2017 19:56:23
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 19:56:23
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)1
__________________________

Log Entry : 
10/18/2017 19:56:23
Error in: . Error Message:entro a Preprocesos 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de AgregarEmbarques0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de CasosSyngenta_y_Acopios0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:entro en PostProcesos 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de SepararAcopiosLDCyACA 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:despues de PostProcesoFacturacion_ReglaExportadores 0
__________________________

Log Entry : 
10/18/2017 19:56:24
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 7979
__________________________

Log Entry : 
10/18/2017 19:56:25
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 7979
__________________________

Log Entry : 
10/18/2017 19:56:25
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 7979
__________________________

Log Entry : 
10/18/2017 19:56:35
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 19:56:35
__________________________

Log Entry : 
10/18/2017 19:56:36
Error in: . Error Message:No hay cartas seleccionadas para facturar
__________________________

Log Entry : 
10/18/2017 20:02:09
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:18:46
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 101154681
__________________________

Log Entry : 
10/18/2017 20:18:49
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 101154681
__________________________

Log Entry : 
10/18/2017 20:18:49
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 101154681
__________________________

Log Entry : 
10/18/2017 20:18:49
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 101154681
__________________________

Log Entry : 
10/18/2017 20:18:49
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 20:18:49
__________________________

Log Entry : 
10/18/2017 20:18:49
Error in: . Error Message:No hay cartas seleccionadas para facturar
__________________________

Log Entry : 
10/18/2017 20:33:30
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 1962867073
__________________________

Log Entry : 
10/18/2017 20:33:32
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 1962867073
__________________________

Log Entry : 
10/18/2017 20:33:32
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 1962867073
__________________________

Log Entry : 
10/18/2017 20:33:32
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 1962867073
__________________________

Log Entry : 
10/18/2017 20:33:32
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 20:33:32
__________________________

Log Entry : 
10/18/2017 20:33:32
Error in: . Error Message:No hay cartas seleccionadas para facturar
__________________________

Log Entry : 
10/18/2017 20:36:08
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 684904144
__________________________

Log Entry : 
10/18/2017 20:37:44
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 684904144
__________________________

Log Entry : 
10/18/2017 20:37:44
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 684904144
__________________________

Log Entry : 
10/18/2017 20:37:44
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 684904144
__________________________

Log Entry : 
10/18/2017 20:37:44
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 20:37:44
__________________________

Log Entry : 
10/18/2017 20:37:44
Error in: . Error Message:No hay cartas seleccionadas para facturar
__________________________

Log Entry : 
10/18/2017 20:39:39
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:44:14
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:48:46
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:49:31
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)1
__________________________

Log Entry : 
10/18/2017 20:49:31
Error in: . Error Message:entro a Preprocesos 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de AgregarEmbarques0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de CasosSyngenta_y_Acopios0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:entro en PostProcesos 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de SepararAcopiosLDCyACA 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:despues de PostProcesoFacturacion_ReglaExportadores 0
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 3832
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 3832
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 3832
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 20:49:32
__________________________

Log Entry : 
10/18/2017 20:49:32
Error in: . Error Message:No hay cartas seleccionadas para facturar
__________________________

Log Entry : 
10/18/2017 20:50:49
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:55:06
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 20:55:58
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)1
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:entro a Preprocesos 0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:System.Data.SqlTypes.SqlTypeException
Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en System.Data.SqlClient.TdsParser.TdsExecuteRPC(SqlCommand cmd, _SqlRPC[] rpcArray, Int32 timeout, Boolean inSchema, SqlNotificationRequest notificationRequest, TdsParserStateObject stateObj, Boolean isCommandProc, Boolean sync, TaskCompletionSource`1 completion, Int32 startRpc, Int32 startParam)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   en System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   en System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   en System.Data.Linq.DataQuery`1.System.Collections.IEnumerable.GetEnumerator()
   en LogicaFacturacion.AgregarEmbarques(List`1& lista, String sc, DateTime FechaDesde, DateTime FechaHasta, Int32 idTitular, Int32 pventa) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 3118
System.Data
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de AgregarEmbarques0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de CasosSyngenta_y_Acopios0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:entro en PostProcesos 0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 0
__________________________

Log Entry : 
10/18/2017 20:56:03
Error in: . Error Message:despues de EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones 0
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:despues de SepararAcopiosLDCyACA 0
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:despues de PostProcesoFacturacion_ReglaExportadores 0
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 4332
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 4332
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 4332
__________________________

Log Entry : 
10/18/2017 20:56:04
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 20:56:04
__________________________

Log Entry : 
10/18/2017 21:03:05
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 21:20:15
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:System.Data.SqlTypes.SqlTypeException
Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en System.Data.SqlClient.TdsParser.TdsExecuteRPC(SqlCommand cmd, _SqlRPC[] rpcArray, Int32 timeout, Boolean inSchema, SqlNotificationRequest notificationRequest, TdsParserStateObject stateObj, Boolean isCommandProc, Boolean sync, TaskCompletionSource`1 completion, Int32 startRpc, Int32 startParam)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   en System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   en System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   en System.Data.Linq.DataQuery`1.System.Collections.IEnumerable.GetEnumerator()
   en Pronto.ERP.Bll.LogicaFacturacion.AgregarEmbarques(List`1& lista, String sc, DateTime FechaDesde, DateTime FechaHasta, Int32 idTitular, Int32 pventa) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 3118
System.Data
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:despues de AgregarEmbarques27
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos27
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor27
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:despues de CasosSyngenta_y_Acopios27
__________________________

Log Entry : 
10/18/2017 21:20:22
Error in: . Error Message:entro en PostProcesos 27
__________________________

Log Entry : 
10/18/2017 21:20:23
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 27
__________________________

Log Entry : 
10/18/2017 21:20:23
Error in: . Error Message:GenerarTablaparamodosnoautomaticos
__________________________

Log Entry : 
10/18/2017 21:20:23
Error in: . Error Message:System.Exception
No se pudo identificar el criterio de agrupación
   en Microsoft.VisualBasic.ErrObject.Raise(Int32 Number, Object Source, Object Description, Object HelpFile, Object HelpContext)
   en Pronto.ERP.Bll.LogicaFacturacion.AgruparItemsDeLaFactura(List`1& oListaCDP, Int32 optFacturarA, String agruparArticulosPor, String SC, String sBusqueda) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 4992
   en Pronto.ERP.Bll.LogicaFacturacion.EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones(List`1& listaDeCartasPorteAFacturar, Int32 optFacturarA, String agruparArticulosPor, String SC, String sBusqueda) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 2945
   en Pronto.ERP.Bll.LogicaFacturacion.PostProcesos(List`1& lista, String optFacturarA, String agruparArticulosPor, String sc) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 1177
   en Pronto.ERP.Bll.LogicaFacturacion.generarTablaParaModosNoAutomaticos(String sc, Object& pag, Object& sesionId, String sLista, String sWHEREadicional, Int64 optFacturarA, String txtFacturarATerceros, String HFSC, String txtTitular, String txtCorredor, String txtDestinatario, String txtIntermediario, String txtRcomercial, String txt_AC_Articulo, String txtProcedencia, String txtDestino, String txtBuscar, String cmbCriterioWHERE, String cmbmodo, String optDivisionSyngenta, String txtFechaDesde, String txtFechaHasta, String cmbPuntoVenta, Int64 startRowIndex, Int64 maximumRows, String txtclienteauxiliar, String& sErrores, String txtFacturarA, String agruparArticulosPor, Object& filas, Object& slinks, String sesionIdposta) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 1440

__________________________

Log Entry : 
10/18/2017 21:20:23
Error in: . Error Message:explota en GetDatatableAsignacionAutomatica
__________________________

Log Entry : 
10/18/2017 21:21:57
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 21:21:57
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 21:21:57
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:System.Data.SqlTypes.SqlTypeException
Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en System.Data.SqlClient.TdsParser.TdsExecuteRPC(SqlCommand cmd, _SqlRPC[] rpcArray, Int32 timeout, Boolean inSchema, SqlNotificationRequest notificationRequest, TdsParserStateObject stateObj, Boolean isCommandProc, Boolean sync, TaskCompletionSource`1 completion, Int32 startRpc, Int32 startParam)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   en System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   en System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   en System.Data.Linq.DataQuery`1.System.Collections.IEnumerable.GetEnumerator()
   en Pronto.ERP.Bll.LogicaFacturacion.AgregarEmbarques(List`1& lista, String sc, DateTime FechaDesde, DateTime FechaHasta, Int32 idTitular, Int32 pventa) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 3118
System.Data
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:despues de AgregarEmbarques27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:despues de CasosSyngenta_y_Acopios27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:entro en PostProcesos 27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 27
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:GenerarTablaparamodosnoautomaticos
__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:System.Exception
No se pudo identificar el criterio de agrupación
   en Microsoft.VisualBasic.ErrObject.Raise(Int32 Number, Object Source, Object Description, Object HelpFile, Object HelpContext)
   en Pronto.ERP.Bll.LogicaFacturacion.AgruparItemsDeLaFactura(List`1& oListaCDP, Int32 optFacturarA, String agruparArticulosPor, String SC, String sBusqueda) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 4992
   en Pronto.ERP.Bll.LogicaFacturacion.EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones(List`1& listaDeCartasPorteAFacturar, Int32 optFacturarA, String agruparArticulosPor, String SC, String sBusqueda) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 2945
   en Pronto.ERP.Bll.LogicaFacturacion.PostProcesos(List`1& lista, String optFacturarA, String agruparArticulosPor, String sc) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 1177
   en Pronto.ERP.Bll.LogicaFacturacion.generarTablaParaModosNoAutomaticos(String sc, Object& pag, Object& sesionId, String sLista, String sWHEREadicional, Int64 optFacturarA, String txtFacturarATerceros, String HFSC, String txtTitular, String txtCorredor, String txtDestinatario, String txtIntermediario, String txtRcomercial, String txt_AC_Articulo, String txtProcedencia, String txtDestino, String txtBuscar, String cmbCriterioWHERE, String cmbmodo, String optDivisionSyngenta, String txtFechaDesde, String txtFechaHasta, String cmbPuntoVenta, Int64 startRowIndex, Int64 maximumRows, String txtclienteauxiliar, String& sErrores, String txtFacturarA, String agruparArticulosPor, Object& filas, Object& slinks, String sesionIdposta) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 1440

__________________________

Log Entry : 
10/18/2017 21:21:58
Error in: . Error Message:explota en GetDatatableAsignacionAutomatica
__________________________

Log Entry : 
10/18/2017 21:23:29
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 21:24:00
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:System.Data.SqlTypes.SqlTypeException
Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en System.Data.SqlClient.TdsParser.TdsExecuteRPC(SqlCommand cmd, _SqlRPC[] rpcArray, Int32 timeout, Boolean inSchema, SqlNotificationRequest notificationRequest, TdsParserStateObject stateObj, Boolean isCommandProc, Boolean sync, TaskCompletionSource`1 completion, Int32 startRpc, Int32 startParam)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   en System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   en System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   en System.Data.Linq.DataQuery`1.System.Collections.IEnumerable.GetEnumerator()
   en Pronto.ERP.Bll.LogicaFacturacion.AgregarEmbarques(List`1& lista, String sc, DateTime FechaDesde, DateTime FechaHasta, Int32 idTitular, Int32 pventa) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 3118
System.Data
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:despues de AgregarEmbarques27
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos27
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor27
__________________________

Log Entry : 
10/18/2017 21:24:29
Error in: . Error Message:despues de CasosSyngenta_y_Acopios27
__________________________

Log Entry : 
10/18/2017 21:24:30
Error in: . Error Message:entro en PostProcesos 27
__________________________

Log Entry : 
10/18/2017 21:24:30
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 27
__________________________

Log Entry : 
10/18/2017 21:25:54
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 21:26:38
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:System.Data.SqlTypes.SqlTypeException
Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en System.Data.SqlClient.TdsParser.TdsExecuteRPC(SqlCommand cmd, _SqlRPC[] rpcArray, Int32 timeout, Boolean inSchema, SqlNotificationRequest notificationRequest, TdsParserStateObject stateObj, Boolean isCommandProc, Boolean sync, TaskCompletionSource`1 completion, Int32 startRpc, Int32 startParam)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader()
   en System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   en System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   en System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   en System.Data.Linq.DataQuery`1.System.Collections.IEnumerable.GetEnumerator()
   en Pronto.ERP.Bll.LogicaFacturacion.AgregarEmbarques(List`1& lista, String sc, DateTime FechaDesde, DateTime FechaHasta, Int32 idTitular, Int32 pventa) en C:\bdl\pronto\BussinessLogic\LogicaFacturacion.vb:línea 3118
System.Data
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:despues de AgregarEmbarques27
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos27
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor27
__________________________

Log Entry : 
10/18/2017 21:26:40
Error in: . Error Message:despues de CasosSyngenta_y_Acopios27
__________________________

Log Entry : 
10/18/2017 21:26:42
Error in: . Error Message:entro en PostProcesos 27
__________________________

Log Entry : 
10/18/2017 21:26:43
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 27
__________________________

Log Entry : 
10/18/2017 22:16:46
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 22:16:55
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 22:16:55
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 22:20:03
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 22:20:03
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 22:20:03
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 22:20:05
Error in: . Error Message:despues de AgregarEmbarques32
__________________________

Log Entry : 
10/18/2017 22:20:05
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos32
__________________________

Log Entry : 
10/18/2017 22:20:05
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor32
__________________________

Log Entry : 
10/18/2017 22:20:05
Error in: . Error Message:despues de CasosSyngenta_y_Acopios32
__________________________

Log Entry : 
10/18/2017 22:20:18
Error in: . Error Message:entro en PostProcesos 32
__________________________

Log Entry : 
10/18/2017 22:20:20
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 32
__________________________

Log Entry : 
10/18/2017 22:23:51
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 22:23:51
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 22:23:52
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 22:23:54
Error in: . Error Message:despues de AgregarEmbarques32
__________________________

Log Entry : 
10/18/2017 22:23:54
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos32
__________________________

Log Entry : 
10/18/2017 22:23:54
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor32
__________________________

Log Entry : 
10/18/2017 22:23:54
Error in: . Error Message:despues de CasosSyngenta_y_Acopios32
__________________________

Log Entry : 
10/18/2017 22:23:57
Error in: . Error Message:entro en PostProcesos 32
__________________________

Log Entry : 
10/18/2017 22:23:57
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 32
__________________________

Log Entry : 
10/18/2017 22:24:00
Error in: . Error Message:despues de EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones 32
__________________________

Log Entry : 
10/18/2017 22:24:00
Error in: . Error Message:despues de SepararAcopiosLDCyACA 32
__________________________

Log Entry : 
10/18/2017 22:24:00
Error in: . Error Message:despues de PostProcesoFacturacion_ReglaExportadores 32
__________________________

Log Entry : 
10/18/2017 22:24:07
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 9871
__________________________

Log Entry : 
10/18/2017 22:24:07
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 9871
__________________________

Log Entry : 
10/18/2017 22:24:07
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 9871
__________________________

Log Entry : 
10/18/2017 22:24:07
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 22:24:07
__________________________

Log Entry : 
10/18/2017 22:31:44
Error in: . Error Message:entrando en GetDatatableAsignacionAutomatica . tanda 0
__________________________

Log Entry : 
10/18/2017 22:31:44
Error in: . Error Message:Cartas sin automatico encontrado (pero este es el modo no automatico!!!)19
__________________________

Log Entry : 
10/18/2017 22:31:45
Error in: . Error Message:entro a Preprocesos 27
__________________________

Log Entry : 
10/18/2017 22:31:47
Error in: . Error Message:despues de AgregarEmbarques32
__________________________

Log Entry : 
10/18/2017 22:31:47
Error in: . Error Message:despues de ExcluirDeGastosAdministrativos32
__________________________

Log Entry : 
10/18/2017 22:31:47
Error in: . Error Message:despues de ReasignarAquellosQueSeLeFacturanForzosamenteAlCorredor32
__________________________

Log Entry : 
10/18/2017 22:31:47
Error in: . Error Message:despues de CasosSyngenta_y_Acopios32
__________________________

Log Entry : 
10/18/2017 22:31:52
Error in: . Error Message:entro en PostProcesos 32
__________________________

Log Entry : 
10/18/2017 22:31:52
Error in: . Error Message:despues de EmparcharClienteSeparadoParaCasosQueSuperenUnMontoDeterminado 32
__________________________

Log Entry : 
10/18/2017 22:31:54
Error in: . Error Message:despues de EmparcharClienteSeparadoParaFacturasQueSuperanCantidadDeRenglones 32
__________________________

Log Entry : 
10/18/2017 22:31:54
Error in: . Error Message:despues de SepararAcopiosLDCyACA 32
__________________________

Log Entry : 
10/18/2017 22:31:54
Error in: . Error Message:despues de PostProcesoFacturacion_ReglaExportadores 32
__________________________

Log Entry : 
10/18/2017 22:32:17
Error in: . Error Message:punto 2 en GetDatatableAsignacionAutomatica . tanda 273
__________________________

Log Entry : 
10/18/2017 22:32:17
Error in: . Error Message:punto 3 en GetDatatableAsignacionAutomatica . tanda 273
__________________________

Log Entry : 
10/18/2017 22:32:17
Error in: . Error Message:punto 4 en GetDatatableAsignacionAutomatica . tanda 273
__________________________

Log Entry : 
10/18/2017 22:32:17
Error in: . Error Message:Separo las facturas que se generan en el lote.18/10/2017 22:32:17
__________________________

Log Entry : 
10/18/2017 22:32:17
Error in: . Error Message:ActualizarCampoClienteSeparador 8 filas
__________________________

Log Entry : 
10/18/2017 22:32:19
Error in: . Error Message:Empiezo a facturar en serio.18/10/2017 22:32:19
__________________________
